plugins/modules/pnp_intent.py

```diff
@@ -953,6 +956,10 @@ class instance for further use.
             return self
 
         update_payload = {"deviceInfo": self.want.get('pnp_params')[0].get("deviceInfo")}
+        if is_stack is True:
+            update_payload["deviceInfo"]["stack"] = True
```

Comment:
Can we check if it is True and assign? Can we simply assign is_stack..

update_payload["deviceInfo"]["stack"] = is_stack

Reply:
Assigned